src/uris/deepLinks/deepLinkService.ts

```diff
@@ -241,9 +241,10 @@ export class DeepLinkService implements Disposable {
 		}
 	}
 
-	private async processPendingDeepLink(pendingDeepLink: StoredDeepLinkContext) {
-		if (pendingDeepLink.url == null) return;
-
+	@debug()
+	private async processPendingDeepLink(pendingDeepLink: StoredDeepLinkContext | undefined) {
+		if (pendingDeepLink?.url == null) return;
+		void this.container.storage.delete('deepLinks:pending');
```

Comment:
```suggestion
		if (pendingDeepLink == null) return;
		void this.container.storage.delete('deepLinks:pending');
		if (pendingDeepLink?.url == null) return;
```

Reply:
This is what I meant. We want to delete the stored value whether or not it has the url property, but we don't want to proceed beyond that without the url.

Reply:
Ah, ok